psycopg/psycopg/connection_async.py

```diff
@@ -434,3 +446,54 @@ async def tpc_recover(self) -> List[Xid]:
             await self.rollback()
 
         return res
+
+
+try:
+    import anyio
```

Comment:
I think it's worth adding an extra so that people can `pip install psycopg[trio]` and get the dependencies too (anyio, sniffio, trio). It would also establish the min required versions.

Reply:
Yes, agreed; I'll do that.

But rather `psycopg[anyio]`, no? Because people might be interested in using AnyIO with the `asyncio` backend. (And we have nothing specific to `trio` on our side.)

Reply:
If you think that `anyio` is a better name I trust that. You know the relation among the projects better than I do :slightly_smiling_face: